docs/DeveloperGuide.md

```diff
@@ -1,34 +1,544 @@
+
+![Sequence of Create Command](images/diagrams/Sequence_CreateCommand.png) <br>
+
+
+#### Remove Command 
+To remove a location, the `RemoveCommand` class is used. The class object is first created by the `Parser` class, where 
+it is then returned to the `Main` class to have its `execute()` method called. The name of the location to be removed 
+will be parsed into the `RemoveCommand` class. The `RemoveCommand` class will call the `removeLocation` method in 
+`LocationList`, which will loop until the name of the location to be removed is found in the `LocationList`. 
+
+Next, the `RemoveCommand` class will call the `deleteByLocation` method in `ApplianceList`, which will loop to look for all the appliance in the location to be removed and delete the appliance. 
+If the name of the location to be removed is not found, the `RemoveCommand` will return the “location does not exist” message.
+
+The sequence diagram for `RemoveCommand` is shown below:
+
+![Sequence of Remove Command](images/diagrams/Sequence_RemoveCommand.png) <br>
+
+
+#### Add Command
+
+To add a new appliance, the `AddCommand` class is used. This class object is first created by the `Parser` class, where 
+it is then returned to the `Main` class to have its `execute()` method called. The name,location,wattage, and type of 
+the appliance will be parsed into the AddCommand class. The `AddCommand` class will perform the `execute()` method to 
+convert the appliance into static (`Fan`, `AirConditioner`, `Lights`, `SmartPlug`) depending on their type. Then the 
+appliance will be added into the ApplianceList. The name of the appliance has to be unique. 
+
+The sequence diagram for `AddCommand` is shown below:
+
+![Sequence of Add Command](images/diagrams/Sequence_AddCommand.png) <br>
+
+#### Delete Command
+
+To delete an appliance, the `DeleteCommand` class is used. This class object is first created by the `Parser` class 
+where it is then returned to the `Main` class to have its `execute()` method called. The name of the appliance to be 
+deleted will be parsed into the `DeleteCommand` class. The `DeleteCommand` class will call the deleteAppliance method 
+in the `ApplianceList` class, which will loop until the name of the appliance to be deleted is found in the 
+`ApplianceList` and remove the appliance from the `ApplianceList`. If the name of the appliance to be deleted is not 
+found, the `DeleteCommand` will return “appliance does not exist” message.
+
+The sequence diagram for `DeleteCommand` is shown below:
+
+![Sequence of Delete Command](images/diagrams/Sequence_DeleteCommand.png) <br>
+
+#### On Command
+
+The `OnCommand` shown below explains the Sequence Diagram of the OnCommand. When the Main class calls the execute() function there will be are 2 cases for on command to flow:
+1. `OnByLocation`
+2. `OnByAppliance`
+
+The sequence diagram for `OnCommand` is shown below:
+
+![Sequence of On Command](images/diagrams/Sequence_OnCommand.png) <br>
```

Comment:
Should the sequence diagrams be separated into different images for easier viewing? Would it be better to use colours?